onnx/defs/controlflow/old.cc

```diff
@@ -181,7 +181,7 @@ ONNX_OPERATOR_SET_SCHEMA(
         .TypeConstraint("B", {"tensor(bool)"}, "Only bool")
         .TypeAndShapeInferenceFunction(IfInferenceFunction));
 
-static const char* Loop_ver16_doc = R"DOC(
+static const char* Loop_ver16_doc_old = R"DOC(
```

Comment:
Hmm. Curious where else this appears?

Reply:
Technically these docs are not “old”. So i wonder if there is a better way to reconcile

Reply:
> Technically these docs are not “old”. So i wonder if there is a better way to reconcile

I didn't compare the strings, one way is to make one of them external and remove another.

Reply:
I think there were mistakes in naming. For example, https://github.com/onnx/onnx/blob/86e0f9527ae32d09f8152aea5c8ec368ee2b8c91/onnx/defs/controlflow/defs.cc#L71 should really be `Loop_ver23_doc`

Reply:
OK, let's check them.